tensorrt_llm/llmapi/llm.py

```diff
@@ -756,32 +816,54 @@ def save(self, engine_dir: str) -> None:
                     f"Copying {file} to {target_engine_dir / file.name}\n")
                 shutil.copy(file, target_engine_dir / file.name)
 
-    def shutdown(self) -> None:
-        if hasattr(self, "_executor") and self._executor is not None:
-            self._executor.shutdown()
-            self._executor = None
 
-        if hasattr(self, 'mpi_session') and self.mpi_session is not None:
-            self.mpi_session.shutdown()
-            self.mpi_session = None
+@append_docstring(TORCH_LLM_DOCSTRING)
+class TorchLLM(BaseLLM):
+    """LLM class is the main class for running a LLM model using PyTorch backend.
 
-    @staticmethod
-    def _shutdown_wrapper(self_ref):
-        # Retrieve the instance if it still exists
-        instance = self_ref()
-        if instance is not None:
-            instance.shutdown()
+    Parameters:
+    """
 
-    def __enter__(self):
-        return self
+    def __init__(self,
+                 model: Union[str, Path],
+                 tokenizer: Optional[Union[str, Path, TokenizerBase,
+                                           PreTrainedTokenizerBase]] = None,
+                 tokenizer_mode: Literal['auto', 'slow'] = 'auto',
+                 skip_tokenizer_init: bool = False,
+                 trust_remote_code: bool = False,
+                 tensor_parallel_size: int = 1,
+                 dtype: str = "auto",
+                 revision: Optional[str] = None,
+                 tokenizer_revision: Optional[str] = None,
+                 **kwargs: Any) -> None:
 
-    def __exit__(self, exc_type, exc_value, traceback) -> bool:
-        del exc_value, traceback
-        self.shutdown()
-        return False  # propagate exceptions
+        kwargs_dict = dict(kwargs)
+        kwargs_dict['backend'] = 'pytorch'
+        super().__init__(model, tokenizer, tokenizer_mode, skip_tokenizer_init,
+                         trust_remote_code, tensor_parallel_size, dtype,
+                         revision, tokenizer_revision, **kwargs_dict)
```

Comment:
`kwargs` is already a dictionary by design:
```suggestion
        kwargs['backend'] = 'pytorch'
        super().__init__(model, tokenizer, tokenizer_mode, skip_tokenizer_init,
                         trust_remote_code, tensor_parallel_size, dtype,
                         revision, tokenizer_revision, **kwargs)
```

Reply:
Nice catch, updated.